src/command/restore/restore.c

```diff
@@ -485,8 +485,11 @@ restoreManifestMap(Manifest *const manifest)
                 // Is this a tablespace?
                 if (target->tablespaceId != 0)
                 {
+                    // Tablespace path without dbid
                     const String *tablespacePath = NULL;
 
+                    LOG_INFO_FMT("using tablespace '%s' in '%s'", strZ(target->name), strZ(target->path));
```

Comment:
Do we need this log?

Reply:
Currently, the log does not show the previous tablespace path. I found this helpful.

Reply:
may be change log with map adding old path to it?

Reply:
Maybe output a new log only for gpdb with information about how it was changed?

Reply:
Does the path change in postgres?

Reply:
The path is changed both in Postgres and GPDB, but GPDB additionally requires the dbid part. I think it's ok to include previous path in the logs for both variants.